docs/documentation/development/bindings/discovery-services.md

```diff
@@ -54,6 +64,7 @@ The following example shows the implementation of the above mentioned methods in
     protected void startBackgroundDiscovery() {
         logger.debug("Start WeMo device background discovery");
         if (wemoDiscoveryJob == null || wemoDiscoveryJob.isCancelled()) {
+            wemoDiscoveryJob = scheduler.scheduleWithFixedDelay(wemoDiscoveryRunnable, 0, refreshInterval, TimeUnit.SECONDS);
```

Comment:
Don't you want to delete the `scheduleAtFixedRate`-line?

Reply:
Of course... Stupid line picking if another line gets changed by accident and you don't want to commit it...